### ISSUE TITLE
Only show Active clients on route lists

### ISSUE THREAD

**Issue by kousu** [savoirfairelinux/sous-chef#771]
### Expected Behaviour

The route list should show only the locations that are relevant to the route.

### Actual Behaviour

There are a lot of clients listed as "Stop: no contact" and "Deceased" that are overwhelming the ability of any user to sort through and actually have the system help them.

Status column's help text is "Configured or unconfigured. Unconfigured clients are placed at the end of the delivery sequence by default." is confusing, too, because it seems like it's talking about sorting the clients into active/unactive, but configured/unconfigured actually seems to be totally orthogonal to that.

### Steps to reproduce

It is only really obvious how much of a problem this is with the larger test database that's behind a password, but if you have that, click here: https://dev.santropolroulant.org/member/route/6/

**Comment by lamontfr** [comment]
@kousu I agree that we should only include active, pending or paused ( see issue #691 ) clients in the route sequence (for example https://sous-chef-test.santropolroulant.org/member/route/3/ ). Too many clients in a route may be the reason that they do not display on the map.

**Comment by kousu** [comment]
@lamontfr, are you on the server right now? That second link is 404'ing for me.

**Comment by lamontfr** [comment]
@kousu No I am not.

**Comment by lamontfr** [comment]
@kousu I fixed most of the issue in PR #776. Regarding the tooltip  "Configured or unconfigured. Unconfigured clients are placed at the end of the delivery sequence by default.", unconfigured means that a new client was added to the route and the customer manager has to position him in the default delivery sequence. I will ask Jean-François for a better term than "configured / unconfigured" that is meaningful in their process. I will leave the issue open for now.
